### DISCUSSION TITLE
Upgrade GO, things to consider?

### DISCUSSION THREAD

**Discussion by jason-barba** [GrandOrgue/grandorgue#876] [General]
I am planning an upgrade of my rather old version of GO. The current verison is fully OK, but there is nearly always good arguments to follow the development of an application.

So, I am browsing the wiki and searching the discussions, but nowhere I find any upgrading guidelines. If there are, please direct me. If there are none, the wiki could do with an article.

//Jens

**Comment by mutti-wilson** [comment]
Make a backup of your settings/combinations and upgrade. You may have to redo some MIDI mapping but it's relatively simple to do.

**Reply by jason-barba** [reply]
Just to make sure: The settings and combinations file(s) is/are stored in my home folder, if I use default path settings? OS=Ubuntu.

A follow up: Do I have to import settings and combinations after upgrade, or are these read by the new GO version on startup?

**Reply by larspalo** [reply]
You can just leave everything as it is. GO will use the same config and settings/cache (if your previous version is not too old == ancient). But it's always better to be safe than sorry, so also make a backup before upgrading to another place and you should be good to go.

**Comment by oleg68** [comment]
> Just to make sure: The settings and combinations file(s) is/are stored in my home folder, if I use default path settings? OS=Ubuntu.

Yes, they are

> A follow up: Do I have to import settings and combinations after upgrade, or are these read by the new GO version on startup?

No, you don't. They are read automatically by GO. But may be some issues with midi device names.
